packages/ramda-extension/src/__tests__/between-test.js

```diff
@@ -0,0 +1,13 @@
+import { between } from '../';
+
+describe('between', () => {
+	it('should return true between numbers', () => {
+		expect(between(1.5, .4, 10)).toEqual(true);
```

Comment:
Just a hint, when you expecting true/false values you should use toBeTruthy/toBeFalsy matchers from jest:
- https://facebook.github.io/jest/docs/en/expect.html#tobetruthy

Reply:
@lukasduspiva actually, toEqual(true) is better than toBeTruthy... toBeTruthy accepts all truthy values such as `1` or `{}`... but we want only `boolean`.. `so toEqual(true)` is far more descriptive